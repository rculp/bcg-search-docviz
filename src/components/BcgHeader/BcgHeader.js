import React from 'react';

import './BcgHeader.css';

const BcgHeader = () => (
  <header className="bcg-header">Minerva</header>
);

export default BcgHeader;
