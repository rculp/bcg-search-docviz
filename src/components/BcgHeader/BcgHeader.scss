@import 'src/global-styles/responsive/utilities';
@import 'src/global-styles/responsive/variables';

@import 'src/global-styles/colors/variables';
@import 'src/global-styles/colors/declarations';

@import 'src/global-styles/typography/variables';

.ui.menu {
  border: 0;
  box-shadow: none;
}

.bcg-header {
  width: 100%;
  height: 56px;
  font-weight: bold;
  text-align: center;

  @media (max-width: map-get($grid-breakpoints, sm)) {
    height: 48px;
  }

  .logo {
    font-family: $font-family-sans;
    font-size: 2.8rem;
    width: 56px;
    height: 56px;
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: $color-brand-dark-green;
    color: $white;
    text-align: center;

    @media (max-width: map-get($grid-breakpoints, sm)) {
      width: 48px;
      height: 48px;
      font-size: 2.2rem;
    }
  }
}
