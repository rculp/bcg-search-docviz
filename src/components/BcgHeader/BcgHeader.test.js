import { shallow } from 'enzyme';

import React from 'react';

import BcgHeader from './BcgHeader';

describe('BcgHeader', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <BcgHeader />
    );
    expect(component).toMatchSnapshot();
  });
});
