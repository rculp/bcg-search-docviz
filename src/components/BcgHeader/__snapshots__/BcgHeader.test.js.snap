// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BcgHeader renders and matches our snapshot 1`] = `
<header
  className="bcg-header"
>
  Minerva
</header>
`;
